### ISSUE TITLE
Include licence in pypi release

### ISSUE THREAD

**Issue by CJ-Wright** [casebeer/python-hkdf#6]
Would it be possible to include the licence in the pypi release? It is causing issues with a conda build.

**Comment by pgajdos** [comment]
In case you want to do a release, please consider also to add also tests.py. It can be used during package checking in the distro.

**Comment by tpetazzoni** [comment]
Yes, please include  the license in the pypi release. Thanks a lot!
